cmds/task/create.go

```diff
@@ -0,0 +1,138 @@
+package task
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/taskcluster/slugid-go/slugid"
+	"github.com/taskcluster/taskcluster-cli/config"
+	tcclient "github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/queue"
+)
+
+var (
+	createCmd = &cobra.Command{
+		Use:     "create <image> <command>",
+		Short:   "Creates and schedules a task, using the specified taskId if given.",
+		PreRunE: checkCreateFlags,
```

Comment:
It's mostly to separate the checking of the flags from the actual execution of the command.

That way, the command can run assuming that the flag was defined and passed in the command line.